tests/test_tti.py

```diff
@@ -124,9 +122,9 @@ def test_tti_staggered(shape):
     configuration['dle'] = 'basic'
     rec2, u2, v2, _ = solver_tti2.forward(kernel='staggered')
 
-    u_staggered1 = u1.data[last, :] + v1.data[last, :]
-    u_staggered2 = u2.data[last, :] + v2.data[last, :]
-
-    res = np.linalg.norm(u_staggered1.reshape(-1) - u_staggered2.reshape(-1))
-    log("DSE/DLE introduces error %2.4e in %d dimensions" % (res, len(shape)))
-    assert np.isclose(res, 0.0, atol=1e-8)
+    res1 = np.linalg.norm(u1.data.reshape(-1) - u2.data.reshape(-1))
+    res2 = np.linalg.norm(v1.data.reshape(-1) - v2.data.reshape(-1))
+    log("DSE/DLE introduces error %2.4e, %2.4e in %d dimensions" %
```

Comment:
since we're changing this, can we drop "DSE/DLE"? we shouldn't assume anything about the cause of the numerical error ...

Reply:
THe DSE/DLE is not for numerical error, this is the test to make sure dependency with openmp are processed correctly.